ssn/ssn_separated/ssn.ttl

```diff
@@ -959,23 +836,19 @@ ssn:SurvivalRange rdf:type owl:Class ;
 ssn:System rdf:type owl:Class ;
   rdfs:subClassOf [ a owl:Restriction ; owl:onProperty sosa:isHostedBy ; owl:allValuesFrom sosa:Platform ] ;
   rdfs:subClassOf [ a owl:Restriction ; owl:onProperty ssn:hasSubSystem ; owl:allValuesFrom ssn:System ] ;
-#  rdfs:subClassOf [ a owl:Restriction ; owl:onProperty ssn:hasDeployment ; owl:allValuesFrom ssn:Deployment ] ;
-#  rdfs:subClassOf [ a owl:Restriction ; owl:onProperty ssn:hasOperatingRange ; owl:allValuesFrom ssn:OperatingRange ] ;
-#  rdfs:subClassOf [ a owl:Restriction ; owl:onProperty ssn:hasSurvivalRange ; owl:allValuesFrom ssn:SurvivalRange ] ;
+  rdfs:subClassOf [ a owl:Restriction ; owl:onProperty ssn:hasDeployment ; owl:allValuesFrom ssn:Deployment ] ;
+  rdfs:subClassOf [ a owl:Restriction ; owl:onProperty ssn:hasOperatingRange ; owl:allValuesFrom ssn:OperatingRange ] ;
+  rdfs:subClassOf [ a owl:Restriction ; owl:onProperty ssn:hasSurvivalRange ; owl:allValuesFrom ssn:SurvivalRange ] ;
   dc:source "http://www.w3.org/2005/Incubator/ssn/" ;
-  rdfs:comment "System is a unit of abstraction for pieces of infrastructure for sensing. A system has components, its subsystems, which are other systems." ; # TODO: update with: "A system may have components" ? 
+  rdfs:comment "System is a unit of abstraction for pieces of infrastructure for Sensing or Actuating. A System may have components, its subsystems, which are other systems." ; 
   rdfs:isDefinedBy "http://www.w3.org/ns/ssn" ;
-  rdfs:label "System" ;
-  rdfs:seeAlso "http://www.w3.org/2005/Incubator/ssn/wiki/SSN_Model#System" .
+  rdfs:label "System" .
 
 
 ###  http://www.w3.org/ns/ssn/SystemLifetime
 ssn:SystemLifetime rdf:type owl:Class ;
                    rdfs:subClassOf ssn:SurvivalProperty ;
-                   rdfs:comment "Total useful life of a sensor or system (expressed as total life since manufacture, time in use, number of operations, etc.)." ;
+                   rdfs:comment "Total useful life of a Sensor, Actuator or System (expressed as total life since manufacture, time in use, number of operations, etc.)." ;
```

Comment:
Total useful life of a System (expressed as total life since manufacture, time in use, number of operations, etc.).